_posts/2019-03-18-speed-up-your-pipeline-using-cache.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title:  Speed up your pipeline using the cache
+date: 2019-03-29
+tags: codebuild aws docker cache continuous-integration
+author: JK Gunnink
+---
+
+Fast feedback in development is an important consideration when deploying software often as
+it helps reduce context switching. When a developer creates some functionality, and wants to test
+and validate the functionality using a CI tool, fast feedback is important to maintain focus.
+
+In a recent project I worked on, one of the tasks I was given was to speed up the time it took for
```

Comment:
This doesn't read well.  Artistic license - you may have been given a task but don't make it look that way.  You're pro-active, you understand fast feedback is important.

Phrase the problem; what was the cause.  Concrete numbers is good.

Builds were taking X minutes, because Y which was bad because Z